src/test/java/io/nats/client/support/JwtUtilsTests.java

```diff
@@ -0,0 +1,121 @@
+package io.nats.client.support;
+
+import org.junit.jupiter.api.Test;
+
+import io.nats.client.NKey;
+
+import static io.nats.client.support.JwtUtils.issueUserJWT;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.time.Duration;
+
+public class JwtUtilsTests {
+
+    @Test
+    public void issueUserJWTSuccessMinimal() throws Exception {
+        NKey userKey = NKey.fromSeed("SUAGL3KX4ZBBD53BNNLSHGAAGCMXSEYZ6NTYUBUCPZQGHYNK3ZRQBUDPRY".toCharArray());
+        NKey signingKey = NKey.fromSeed("SAANJIBNEKGCRUWJCPIWUXFBFJLR36FJTFKGBGKAT7AQXH2LVFNQWZJMQU".toCharArray());
+        String accountId = "ACXZRALIL22WRETDRXYKOYDB7XC3E7MBSVUSUMFACO6OM5VPRNFMOOO6";
+        String jwt = issueUserJWT(signingKey, accountId, new String(userKey.getPublicKey()), null, null, null, 1633043378);
+        String cred = String.format(JwtUtils.NATS_USER_JWT_FORMAT, jwt, new String(userKey.getSeed()));
+        /*
+            Generated JWT:
+            {
+                "iat": 1633043378,
+                "jti": "X4TWD4I3YGCAZ4PRKULOOUACR5W5AIKSM6MXVPJZBLXHZZMNFZAQ",
+                "iss": "ADQ4BYM5KICR5OXDSP3S3WVJ5CYEORGQKT72SVRF2ZDVA7LTFKMCIPGY",
+                "name": "UA6KOMQ67XOE3FHE37W4OXADVXVYISBNLTBUT2LSY5VFKAIJ7CRDR2RZ",
+                "nats": {
+                    "issuer_account": "ACXZRALIL22WRETDRXYKOYDB7XC3E7MBSVUSUMFACO6OM5VPRNFMOOO6",
+                    "type": "user",
+                    "version": 2
+                },
+                "sub": "UA6KOMQ67XOE3FHE37W4OXADVXVYISBNLTBUT2LSY5VFKAIJ7CRDR2RZ"
+            }
+         */
+        assertEquals("-----BEGIN NATS USER JWT-----\n" +
+            "eyJ0eXAiOiJKV1QiLCAiYWxnIjoiZWQyNTUxOS1ua2V5In0.eyJpYXQiOjE2MzMwNDMzNzgsImp0aSI6Ilg0VFdENEkzWUdDQVo0UFJLVUxPT1VBQ1I1VzVBSUtTTTZNWFZQSlpCTFhIWlpNTkZaQVEiLCJpc3MiOiJBRFE0QllNNUtJQ1I1T1hEU1AzUzNXVko1Q1lFT1JHUUtUNzJTVlJGMlpEVkE3TFRGS01DSVBHWSIsIm5hbWUiOiJVQTZLT01RNjdYT0UzRkhFMzdXNE9YQURWWFZZSVNCTkxUQlVUMkxTWTVWRktBSUo3Q1JEUjJSWiIsIm5hdHMiOnsiaXNzdWVyX2FjY291bnQiOiJBQ1haUkFMSUwyMldSRVREUlhZS09ZREI3WEMzRTdNQlNWVVNVTUZBQ082T001VlBSTkZNT09PNiIsInR5cGUiOiJ1c2VyIiwidmVyc2lvbiI6Mn0sInN1YiI6IlVBNktPTVE2N1hPRTNGSEUzN1c0T1hBRFZYVllJU0JOTFRCVVQyTFNZNVZGS0FJSjdDUkRSMlJaIn0.CUo-0aCxS1DvPpvZ3cZMXmLlQRMKYRfQjjXSFiCkmRrIJKX9crdkcIBuoYIgfczt5oTEDyR1ldS_ucipxCnoAQ\n" +
```

Comment:
just to be sure, the `\n` at the end is an artefact of the test, not the jwt.
I stored this string without the `\n` and nsc is happy. 
```
nsc describe jwt -f x
+---------------------------------------------------------------------------+
|                                   User                                    |
+----------------+----------------------------------------------------------+
| Name           | UA6KOMQ67XOE3FHE37W4OXADVXVYISBNLTBUT2LSY5VFKAIJ7CRDR2RZ |
| User ID        | UA6KOMQ67XOE3FHE37W4OXADVXVYISBNLTBUT2LSY5VFKAIJ7CRDR2RZ |
| Issuer ID      | ADQ4BYM5KICR5OXDSP3S3WVJ5CYEORGQKT72SVRF2ZDVA7LTFKMCIPGY |
| Issuer Account | ACXZRALIL22WRETDRXYKOYDB7XC3E7MBSVUSUMFACO6OM5VPRNFMOOO6 |
| Issued         | 2021-09-30 23:09:38 UTC                                  |
| Expires        |                                                          |
| Issuer Scoped  | Yes                                                      |
+----------------+----------------------------------------------------------+
```